insights/util/component_graph.py

```diff
@@ -100,6 +100,7 @@ def main(filename):
     dr.load_components("insights.specs.default")
     dr.load_components("insights.specs.insights_archive")
     dr.load_components("insights.specs.sos_archive")
+    dr.load_components("insights.specs.jdr_archive")
```

Comment:
Also, please add this line to `insights/tools/specs_to_rules.py`.

Reply:
@bfahr 
I checked the specs_to_rules.py, we do not load jdr_archive, sos_archive and insights_archive. It works well.

Reply:
You are correct, sorry about that.